test/specs/with-options.spec.js

```diff
@@ -45,7 +48,7 @@ import test from './test/fixtures/icons.svg';`);
       plugins: [["./lib/index.js", { disableSVGO: true }]],
     });
 
-    expect(code).toBe(expectedRaw);
+    expect(code.replace(/\\r/gi, "")).toBe(expectedRaw);
```

Comment:
This seems unnecessary?

Reply:
I did this because running in windows, the tests wasn't passing due to windows put a \r in the line breaks...

Reply:
Ohh, I see. That's annoying.

We _could_ make the test platform-agnostic by importing the specific EOL character and using it:

```js
const { EOL } = require('os');
...
const someRaw = `blah blah svg${EOL} some more SVG`
```

But if this is the only test affected, maybe this regex is fine. If we do stick with the regex though, can you add a little comment about why it's there?